infrastructure/environments/setup-environment.ts

```diff
@@ -718,6 +718,25 @@ const derivedVariables = [
   }
 ] as const
 
+const metabaseAdminQuestions = [
+  {
+    valueType: 'VARIABLE',
```

Comment:
```suggestion
    valueType: 'SECRET',
```
let's keep it as a SECRET variable as we are doing so for the other usernames/emails.